bot_control.py

```
@@ -0,0 +1,110 @@
+from typing import Dict, Optional, List
+import discord
```

Flake8 (7.2.0)

[error] 'discord' imported but unused

(F401)